### ISSUE TITLE
var always fails

### ISSUE THREAD

**Issue by szepeviktor** [douglascrockford/JSLint#175]
```js
var rx_amp = /&/g,
    rx_gt = />/g,
    rx_lt = /</g;
```
JSLint says: `Expected ';' and instead saw ','.`
http://www.jslint.com/

Could you fix it?

**Comment by douglascrockford** [comment]
Use the multivar option.

**Comment by szepeviktor** [comment]
Thank you.
I write instead
```js
var rx_amp = /&/g;
var rx_gt = />/g;
var rx_lt = /</g;
```
